cmake/tools.cmake

```diff
@@ -160,6 +160,10 @@ function(project_build_tools_patch_default_imported_config)
       if(IS_ALIAS_TARGET)
         continue()
       endif()
+      get_target_property(IS_ALIASED_TARGET ${TARGET_NAME} ALIASED_TARGET)
+      if (IS_ALIASED_TARGET)
+        continue()
+      endif()
```

Comment:
This `if` condition appears redundant, as it checks the same `ALIASED_TARGET` property again

Reply:
Adding reply from @tjcw over mail - No, one is ALIAS and the other is ALIASED

Reply:
@tjcw maybe I am missing something here, both variables `IS_ALIASED_TARGET` and `IS_ALIAS_TARGET` are checking for the same property `ALIASED_TARGET`. So what exactly is the difference?

Reply:
I will try and find the colleague who wrote that code and ask him or her.

Reply:
I asked my colleague. He thinks we applied a local fix, then you (upstream) applied a practically-equivalent fix and so we ended up with both in a patch. I have removed the duplicate.